jest.config.js

```diff
@@ -13,6 +13,6 @@ export default {
     'src/types',
   ],
   reporters: ['default'],
-  globals: { 'ts-jest': { diagnostics: false } },
+  globals: { 'ts-jest': { diagnostics: false, isolatedModules: true } },
```

Comment:
We care about TypeScript checks in our application. We don't care about TypeScript errors in our tests. We just care that they pass.